### PR TITLE
Remove default_formula requirements.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
This matched the similar work we've done for `depends_on :foo` stuff.

### CODE REVIEW

File: Formula/wget.rb [hunk 1]
```diff
@@ -6,6 +6,8 @@ class Wget < Formula
   sha256 "4f4a673b6d466efa50fbfba796bd84a46ae24e370fa562ede5b21ab53c11a920"
   revision 1
 
+  env :std
```
Comment (DomT4) [inline Formula/wget.rb]: What's going on with `wget` here?

Comment (MikeMcQuaid) [reply Formula/wget.rb]: @DomT4 Nothing to see here, totally didn't push an unrelated change 😆

Comment (DomT4) [reply Formula/wget.rb]: 😸 I spent a minute reading through the `wget` formula after this PR popped up like I'm sure there's no requirement there 🙈.

Comment (MikeMcQuaid) [reply Formula/wget.rb]: @DomT4 Was my testing https://github.com/Homebrew/brew/pull/3650

Comment (DomT4) [reply Formula/wget.rb]: Ah 👍. Seems like a reasonable change.

File: Formula/zpython.rb [hunk 2]
```diff
@@ -1,20 +1,3 @@
...
-  end
-
-  def message
-    "Zsh 5.x is required to install. Consider `brew install zsh`."
-  end
-end
-
 class Zpython < Formula
   desc "Embeds a Python interpreter into zsh"
   homepage "https://bitbucket.org/ZyX_I/zsh"
```
Comment (vinyldarkscratch) [inline Formula/zpython.rb]: Hey there, seems like this site is down for the count, can we replace it with http://www.zsh.org/?

**Review by queengooborg — CHANGES_REQUESTED** [review]

File: Formula/zpython.rb (hunk 2 above)
Comment (fxcoudert) [reply Formula/zpython.rb]: It's a network fluke on CI, works fine here.

**Review by fxcoudert — APPROVED** [review]

File: Formula/zpython.rb (hunk 2 above)
Comment (vinyldarkscratch) [reply Formula/zpython.rb]: Ah, never mind, it seems like it was actually something with the BitBucket user -- I tried connecting earlier, but I got a 404 on my own end.  Never mind!

**Pull request merged**